db/views/reporting_facility_state_daily_dimensions_v01.sql

```diff
@@ -0,0 +1,222 @@
+WITH follow_up_blood_pressures AS (
+    SELECT DISTINCT ON (patient_id, facility_id, day_of_year)
+        p.id AS patient_id,
+        p.gender::gender_enum as patient_gender,
+        bp.id as visit_id,
+        'BloodPressure' AS visit_type,
+        bp.facility_id,
+        bp.user_id,
+        bp.recorded_at AS visited_at,
+        cast(EXTRACT(DOY FROM bp.recorded_at AT TIME ZONE 'UTC' at time zone (SELECT current_setting('TIMEZONE'))) as integer) AS day_of_year
+    FROM patients p
+             INNER JOIN blood_pressures bp
+                        ON p.id = bp.patient_id
+                            AND date_trunc('day', bp.recorded_at AT TIME ZONE 'UTC' AT TIME ZONE (SELECT current_setting('TIMEZONE')))
+                               > date_trunc('day', p.recorded_at AT TIME ZONE 'UTC' AT TIME ZONE (SELECT current_setting('TIMEZONE')))
```

Comment:
You can use something like https://sqlformat.org to indent this consistently, Rubymine does a bad job at times.

Reply:
okay